lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -642,7 +654,7 @@ def mock_run(args, **_kwargs):
                 args, returncode=0, stdout="Successfully Unpacked!", stderr=None
             )
 
-        def mock_resolve(_path, _strict=False):
+        def mock_resolve(_path, strict: bool = False):
             """In this scenario, there are no symlinks,
             so resolve() should never be called."""
             raise AssertionError("Unexpected call to Path.resolve()")
```

Comment:
The `strict` parameter is never used, so it should retain its leading underscore.

Reply:
No, it shouldn't, because Python doesn't like a call with `strict=True` when there's no `strict` argument defined.